### DISCUSSION TITLE
How to cancel Enterprise Cloud Trial

### DISCUSSION THREAD

**Discussion by mye-creator** [community/community#29070] [Enterprise] [answered]
Hi, sorry for bothering you just for that stupid question but how to cancel the trial of Enterprise Cloud Trial? I didn't mean to actually buy enterprise status for the organization. Can someone help me to cancel it ASAP?

**Comment by GabyCT** [comment, marked as answer]
Here it is the info https://docs.github.com/es/get-started/signing-up-for-github/setting-up-a-trial-of-github-enterprise-cloud#finishing-your-trial

**Reply by queenofcorgis** [reply]
Thanks @GabyCT for posting the instructions, @mye-creator if you follow those steps you should be able to cancel on your own.

**Reply by BenitzCoding** [reply]
Unfortunately, I can't find the button. Can I get some help?
This is what I see:
<img width="947" alt="image" src="https://user-images.githubusercontent.com/70798458/200541690-1154917c-c2ca-4af6-a937-c32a46156fd7.png">

**Reply by eosvn** [reply]
The same problem. I can't see the Cancel button.

**Reply by apvgroups** [reply]
follow these stepts

Go to GitHub → Your profile → Enterprises
Open your Enterprise account
Click Settings
Go to General
Scroll down to “Danger Zone”
Click Cancel trial (or Delete trial if already expired)
Confirm the action
